src/ensembl/src/content/app/entity-viewer/components/transcript-visualisation/TranscriptVisualisation.scss

```diff
@@ -0,0 +1,7 @@
+.containerSvg {
+  overflow: visible;
+}
+
+.emptyBlock {
+  fill: none !important;
```

Comment:
Normally, I am dead set against `!important`, but in this case it looks like the only mechanism to override fill property that might be coming from the parent (it doesn't make sense for the parent to also pass a custom `emptyBlock` class, because empty blocks will always have the same `fill: none` rule. Either use `!important` in a css file, or an inline style rule in a react component.